tasks/config.yml

```diff
@@ -290,6 +290,6 @@
   become: true
   copy:
     src: "{{ bbb_custom_presentation }}"
-    dest: "/var/www/bigbluebutton-default/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
+    dest: "{{ bbb_nginx_root | regex_replace('\\/$', ''}}/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
```

Comment:
I think the `regex_replace` function is missing the closing parenthesis `)`

And just to confirm this regex function is needed to remove the trailing `/`, right?

Reply:
Thank you for the review @francisco-core .
I fixed it in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/bef7c63da4f638dc42669b3fa78460c6bf64fc35